### PR TITLE
fix: spring boot and more

### PR DESCRIPTION
**Pull request by wurstbrot**

### CODE REVIEW

File: src/main/java/org/owasp/dsomm/metricCA/analyzer/MetricAnalyzerApplication.java
Comment (sumsumcity) [inline src/main/java/org/owasp/dsomm/metricCA/analyzer/MetricAnalyzerApplication.java:1]: @wurstbrot I have trouble to run it. It's strange anyway why it works for you, as there are errors in the import of the files. You import from analyzer which does not exist anymore. You renamed it to metricAnalyzer.

File: src/main/java/com/analyzer/App.java
Comment (sumsumcity) [inline src/main/java/com/analyzer/App.java:1]: @wurstbrot Makes sense that you delete it. Should now the yaml reader etc work in the new main when starting the SpringApplication?

**Review by sumsumcity — COMMENTED** [review]
@wurstbrot Please check my comments.

File: src/main/java/com/analyzer/App.java
Comment (wurstbrot) [reply src/main/java/com/analyzer/App.java:1]: yes

File: src/main/java/org/owasp/dsomm/metricCA/analyzer/MetricAnalyzerApplication.java
Comment (wurstbrot) [reply src/main/java/org/owasp/dsomm/metricCA/analyzer/MetricAnalyzerApplication.java:1]: I don't get your comment.
The package of this file is `org.owasp.dsomm.metricCA.metricAnalyzer`. which is wrong. I changed it to `org.owasp.dsomm.metricCA.analyzer`. The others are in `org.owasp.dsomm.metricCA.analyzer.*` and that is correct.

**Pull request merged**